### PR TITLE
[WebProfilerBundle] Minor plural/singular change

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| License | MIT |
| Fixed tickets | - |

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig [hunk 1]
```diff
@@ -144,7 +144,7 @@
                         tbody.appendChild(rows);
 
                         if (infoSpan) {
-                            var text = requestStack.length + ' calls';
+                            var text = requestStack.length + ' call' + (reqeustStack.length > 1 ? 's' : '');
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig]: typo in the variable name

Comment (WouterJ) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig]: Thanks, fixed now. (made that same typo 3 times yesterday...)

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Good catch, thanks @WouterJ.

**Pull request merged**